you match salt clients by minion_id or with wildcards (#670)
SUSE/doc-susemanager: modules/reference/pages/salt/salt-remote-commands.adoc

On the menu:Salt[Remote Commands] page you will see two text boxes.
The first box is for entering commands.

> The second box is for targeting clients by the [literal]``minion_id`` value (name), or by a partial match using wildcards.

Reviewer: Can we change this to be the field names ("in the [guimenu]``foo`` field"), rather than first and second boxes?

Author: As said, these boxes don't have labels.  I'm clueless...

Reviewer: grrr

Reviewer 2: > As said, these boxes don't have labels. I'm clueless...

You are right. They are `pre filled` by default, but if you delete the content you have no idea what they stand for. :disappointed:

> Input a command you want to execute, then add a target client or a set of clients (using wildcard matching) where you want to execute the command on.

Reviewer: ```suggestion
Enter the command you want to execute.
Add a target client to execute the command on, or use wildcard matching to target a group of clients.
```

> Select the btn:[Find Targets] button to verify which clients will be targeted.

Reviewer: ```suggestion
Click btn:[Find Targets] to verify which clients will be targeted.
```

> Select the btn:[Run Command] button to execute a command on the selected systems.

Reviewer: ```suggestion
Click btn:[Run Command] to execute the command on the target systems.
```
